### ISSUE TITLE
`concatenate!` macro cannot expand to public type

### ISSUE THREAD

**Issue by graelo** [vks/average#19]
Hi,

I'd like to use `concatenate!()` and obtain a a public type, but I could not find a way.
Do you envision a simple way to do this?

Thanks

**Comment by vks** [comment]
The easiest workaround would be to use the newtype pattern to define a public type:

```rust
concatenate!(PrivateEstimator, ...);
pub struct PublicEstimator(PrivateEstimator);
```

However, this will result in quite a bit of boilerplate code. I'll consider extending `concatenate!` to support creating a public type.

Sorry for the late reply!

**Comment by jellythefish** [comment]
@vks hey! Would be great to have this feature as well =)
It seems like @hatoo's pr https://github.com/vks/average/pull/28 do the right thing. Could you please check?

**Comment by vks** [comment]
I just published version 0.15.1, which includes this feature.
